src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -78,6 +85,27 @@ public JavaGapicSurfaceTestTransformer(GapicCodePathMapper javaPathMapper) {
     return views;
   }
 
+  public Iterable<Interface> getGrpcInterfacesToMock(Model model, ApiConfig apiConfig) {
```

Comment:
Should this be public or private? If this is public method some comments would be helpful.